subgraph/schema.graphql

```diff
@@ -31,6 +31,8 @@ type Lock @entity {
   lastKeyMintedAt: BigInt
   "Address of the lock deployer"
   deployer: Bytes!
+  "Receipts number of lock"
+  receiptCount: BigInt!
```

Comment:
This is wrong. How can you keep track of the receipt count if you don't have a way to count them??